install_prerequisites.py

```diff
@@ -82,18 +82,26 @@ def main():
             'zlib1g-dev',
         ]
 
+        apt_env = os.environ.copy()
+        apt_env['DEBIAN_FRONTEND'] = 'noninteractive'
+
         if nfpm_path is None:
             repo_list_path = '/etc/apt/sources.list.d/goreleaser.list'
 
             if not os.path.exists(repo_list_path):
+                # install apt-transport-https and ca-certificates first
+                cmd = ['apt-get', 'update', '-y']
+                build.run_cmd(cmd, check_rc='getting updates failed')
+
+                cmd = ['apt-get', 'install', '-y', 'ca-certificates']
```

Comment:
Should the comment not be mentioning `apt-transport-https`?

Reply:
No. I was originally installing `apt-transport-https`, but it's a transitional/dummy package these days.
Fixed.